app/http.php

```diff
@@ -199,8 +200,34 @@ function logError(Throwable $error, string $action, ?Logger $logger, Utopia\Rout
             throw new Exception('No online executor found', 404);
         }
 
+        /**
+         * @var string $hostname
+         */
         $hostname = $option->getState('hostname') ?? '';
 
+        // Optimistic update. Mark runtime up instantly to prevent race conditions
+        // Next health check with confirm it started well, and update usage stats
+        $runtimeId = $request->getHeader('x-opr-runtime-id', '');
```

Comment:
Can we try to get the $runtimeId from the URL path?

Reply:
Keeping it as header so we can move quickly.